typebinder/src/exporters/file.rs

```diff
@@ -42,13 +45,35 @@ impl FileExporter {
                 .expect("Invalid UTF-8 name for module")
         });
     }
+
+    pub fn set_header_comment(&mut self, header_comment: HeaderComment) {
+        self.header_comment = header_comment;
+    }
+
+    fn render_header_comment(&self, rust_module_path: &syn::Path) -> Option<String> {
+        match &self.header_comment {
+            HeaderComment::None => None,
+            HeaderComment::Custom(comment) => Some(format!("// {}", comment)),
```

Comment:
Custom comments may want to be multi-line as well. Pick one option : 

* We let the user set the comment mark `//` themselves on all lines
* We do `comment.lines()` and prepend a comment mark `//` on all lines
* We add `/*` before and `*/` after so the comment can be multi-line as the user wishes

I think option 3 is better.